packages/recomponents/src/components/r-popper/r-popper.vue

```diff
@@ -194,8 +194,10 @@
                 this.isPopperVisible = visible;
                 await this.$nextTick(); // await for contentEl to be visible before trying to access it
                 if (this.contentEl && this.contentEl.hasChildNodes()) {
-                    this.contentEl.firstChild.style = `transition-duration: ${this.duration}s`;
-                    this.positionContent();
+                    if (this.contentEl.firstChild.classList) {
```

Comment:
We could avoid nested `if` statements by adding one more condition:
```js
if(this.contentEl && this.contentEl.hasChildNodes() && this.contentEl.firstChild.classList) {
    // ...
}
```

Reply:
LOL, Indeed, thank you!